### ISSUE TITLE
[Feature Request] Disable Vulkan Queue Scheduler completely on Nvidia GPUs.

### ISSUE THREAD

**Issue by Hasster1** [RPCS3/rpcs3#11582]
## Quick summary
If the fast option is internally disabled, then why not disable the whole option itself? Kinda confuses me. 
## Details

It just confused me a little bit. Other settings, that i can't change because of my CPU, are grayed out all the time, while this one isn't. Probably because it's a GPU option, and not a CPU?

#### 1. Please describe, what part of RPCS3 would be affected by your feature:
* UI
#### 2. Please tell us, why your feature is important to RPCS3.
It's not really important, just something that's bugging me.
#### 3. Please attach screenshots of the feature implemented in other projects.
![image](https://user-images.githubusercontent.com/20757715/155831294-14957184-0785-44e6-b253-934876e5f91b.png)
Something like that.
#### 4. Please provide your system configuration:
* OS - Windows 10 1909
* CPU - i5-4210u.
* GPU - Nvidia GeForce 840m
* Driver version - 511.79

**Comment by kd-11** [comment]
> It just confused me a little bit. Other settings, that i can't change because of my CPU, are grayed out all the time, while this one isn't. Probably because it's a GPU option, and not a CPU?

Yes, because it has to be dynamic, it is common to have multiple GPUs or multiple GPU drivers, especially outside windows so we cannot just disable the option if the user happens to own an NVIDIA card. There is no technical limitation here, just that it requires extra work for little benefit. The "fast" option will be dropped down the line, its there to test for regressions in the short term. It makes more sense to just hide the option entirely or move it to the debug tab.
